tests/platforms/windows/app/test_package.py

```diff
@@ -27,12 +28,50 @@ def package_command(tmp_path):
         version="81.2.1.0",
         arch="groovy",
     )
+    command.tmp_path = tmp_path
```

Comment:
This isn't needed - `tmp_path` is guaranteed to be consistent within a given test. You don't need to annotated it onto the command.

Reply:
OK, got it. These pytest fixtures a still enigmatic to me.